### ISSUE TITLE
Electronic Hardening - Effect is not simulated in Pyfa

### ISSUE THREAD

**Issue by TheTheladio** [pyfa-org/Pyfa#2602]
Dear Ladies and Gentlemen, 

with the search for 'Electronic Hardening' here on the page, I have found 3 issues (all fixed) from 2019 and 2017.

Today I found a bug regarding that charge again. 
When simulating the effect of that charge on my current ship, or via projection onto another ship, it only simulates the bonus for Sensor Strength (18%), but not the 'Remote Sensor Dampener / Remote Weapon Disruption Resistance bonus' (-9%), when said ship is effected with a TD or a RSD in the 'Projected' tab below. 

I've used the following Abos fitting to test it:
++++
[Absolution, Absolution EH Test]

[Empty Low slot]
[Empty Low slot]
[Empty Low slot]
[Empty Low slot]
[Empty Low slot]
[Empty Low slot]
[Empty Low slot]

[Empty Med slot]
[Empty Med slot]
[Empty Med slot]

Heavy Beam Laser II
Information Command Burst II, Electronic Hardening Charge
[Empty High slot]
[Empty High slot]
[Empty High slot]
[Empty High slot]
[Empty High slot]

[Empty Rig slot]
[Empty Rig slot]

Information Command Mindlink
++++

In the 'Projected' Tab I've added the following:
Remote Sensor Dampener II x1
Tracking Disruptor II x1
(both active)

Suppose you now switch the Information Command Burst II on and off. In that case, you will see that the Sensor Strength bonus will be applied, but not the 'Remote Sensor Dampener / Remote Weapon Disruption Resistance bonus' on the Heavy Beam Optimal Range nor Falloff, neither on the Ship Targeting Range nor Scan Resolution. 

![image](https://github.com/pyfa-org/Pyfa/assets/166345074/03a29d22-c7d5-454c-84f3-16aa8227c6d4)

Please let me know if questions remain, and THANK YOU for an otherwise awesome and very helpful tool!!!

Yours sincerely 
Theladio :)

**Comment by lunedis** [comment]
Might have something to do with you using raw modules in the projected tab instead of projecting whole ship fittings.

I just tested it using a Crucifier fitting, and while I did not factcheck the actual numbers, the result seems to be working.

Abso base range: 33+10

Abso range with 2 TDs: 6.6+2

Abso range with 2 TDs and EH link: 11.4+3.4

Targeting range also changes accordingly after applying a maulus.

**Comment by DarkFenX** [comment]
> However, I still believe that it should work with Raw Modules as well.

I agree it should. Even if it doesn't work, I am not going to fix it. Projection code in pyfa is a hot mess and I'd prefer not to touch it.

(and as usual, i have replacement calculation engine being cooked, so rather put my spare time into it)
